src/server.c

```diff
@@ -183,885 +183,891 @@ struct redisServer server; /* Server global state */
 struct redisCommand redisCommandTable[] = {
     {"module",moduleCommand,-2,
      "admin no-script",
-     0,NULL,0,0,0,0,0,0},
+     0,NULL,0,0,0,0,0,0,0},
```

Comment:
Agree :) Will do that change - since yesterday we already had two conflicts =) )

Reply:
we have a plan for the future, but i'm also hoping to find a plan for now (would very much like to avoid these modifications)

Reply:
I think we can just omit that change for now, and it'll all pass.
it means that the last `0` which was the command `id` will now be stored into the new `failed_calls` and the last member (the `id`) will be implicitly initialized to 0 (which is ok).

i remember in the past that some compiler generated a warning on structs that were partially initialized but not fully initialized, but i see in the CI for `#8210` passes and we do use `-Werror`.
ideally that means we can trim the last 3 `0`, and not just avoid adding a 4th, but i don't wanna do that.

i'm planning another huge PR soon which will completely rewrite that whole command table array, so that's when i'll trim these.

Reply:
@oranagra followed the suggestion and removed this changes ( it makes the PR a lot cleaner )